src/Testcontainers/Configurations/WaitStrategies/HttpWaitStrategy.cs

```diff
@@ -38,6 +38,8 @@ public sealed class HttpWaitStrategy : IWaitUntil
 
     private HttpMessageHandler _httpMessageHandler;
 
+    private Func<HttpContent> _contentFactory = () => null;
```

Comment:
Initializing the default value is done via the default constructor and the corresponding with member: https://github.com/testcontainers/testcontainers-dotnet/blob/55018dc890f70ca70c0440002c0561c5bb122175/src/Testcontainers/Configurations/WaitStrategies/HttpWaitStrategy.cs#L48

It would be nice if we can use the same approach for the HTTP content too.

Reply:
sure 👍 , I aligned it: https://github.com/testcontainers/testcontainers-dotnet/pull/963/commits/244a873750a660102a0393423e1d924f07f43940